lib/smart.js

```diff
@@ -46,7 +47,7 @@ function authFromCapability(capabilityStatement) {
     });
     return authMetadata;
   } catch (error) {
-    console.log(error); // eslint-disable-line no-console
+    logError(error);
     return authMetadata;
```

Comment:
Should this error be rethrown? This method doesn't make any http requests, so I'm guessing this `catch` exists to handle capability statements that are malformed or don't contain the keys we expect. In such cases would it be better to surface an error to the user or not?

Reply:
I think as long as we get an empty object back if the section doesn't exist (e.g. the server doesn't have SMART support). It would be nice to have an info log about the status of SMART URLs being there or not.

Reply:
In other words, I don't think we should raise here if there are nulls, we should return `{}`.

Reply:
At some point we may want to refactor this so that catching an error isn't part of the normal control flow. It sure would be nice if js had safe navigation...